app/views/doc/_ext_books.erb

```diff
@@ -40,11 +40,18 @@
       </li>
       <li>
         <a href="https://www.packtpub.com/application-development/git-essentials-second-edition"><img src="https://www.packtpub.com/sites/default/files/B06881.png" width="113"/></a>
-        <h4><a href="https://www.packtpub.com/application-development/git-essentials-second-edition">Git Essentials - Second Edition</a></h4>
+        <h4><a href="https://www.packtpub.com/application-development/git-essentials-second-edition">Git Version Control Cookbook - Second Edition</a></h4>
```

Comment:
why are we changing the title of this book? The url still points to a book called "Git Essentials - Second edition"